metron-deployment/packaging/docker/rpm-docker/SPECS/metron.spec

```diff
@@ -47,7 +47,8 @@ Source2:        metron-elasticsearch-%{full_version}-archive.tar.gz
 Source3:        metron-data-management-%{full_version}-archive.tar.gz
 Source4:        metron-solr-%{full_version}-archive.tar.gz
 Source5:        metron-enrichment-%{full_version}-archive.tar.gz
-Source6:        metron-pcap-backend-%{full_version}-archive.tar.gz
+Source6:        metron-indexing-%{full_version}-archive.tar.gz
```

Comment:
So, is the Source${x} naming convention imposed by spec?  Seems like we might have the opportunity to name things more descriptively.  I don't mind it for this PR and it's not a big thing..I'm just curious.